Google.Api.Gax/Platform.cs

```diff
@@ -98,6 +96,30 @@ private static async Task<string> LoadMetadataAsync()
             }
         }
 
+        /// <summary>
+        /// Determine the metadata host to use. In order of priority, this will use:
+        /// - GCE_METADATA_HOST environment variable, if set and non-empty
+        /// - METADATA_EMULATOR_HOST environment variable, if set and non-empty.
```

Comment:
I would have thought the emulator comes first.

Reply:
Confirmed that GCE_METADATA_HOST should take priority. Merging.